shared/constants/teams.js

```diff
@@ -84,6 +85,19 @@ const getTopicFromConvID = (state: TypedState, conversationIDKey: ChatTypes.Conv
 const getRole = (state: TypedState, teamname: Types.Teamname): ?Types.TeamRoleType =>
   state.entities.getIn(['teams', 'teamNameToRole', teamname], null)
 
+const getCanPerform = (state: TypedState, teamname: Types.Teamname): RPCTypes.TeamOperation =>
+  state.entities.getIn(['teams', 'teamNameToCanPerform', teamname], {
+    manageMembers: false,
```

Comment:
:sheep: this obj could be factored out into a separate constant `initialCanUserPerform`, and then could be typed